src/main/java/com/google/step/snippet/external/W3SchoolsClient.java

```diff
@@ -57,7 +57,17 @@ public Card search(String w3Link, String query) {
       return null;
     }
     String title = Jsoup.clean(titles.first().text(), Whitelist.relaxed());
-    String description = Jsoup.clean(descriptions.first().text(), Whitelist.relaxed());
+    String description;
+    /* If description for specific code example exists, use example description, otherwise use first
+    page description */
```

Comment:
Nit: I would suggest using double-slash comments for comments inside your methods and add a period at the end of the sentence.

```
// If description for specific code example exists, use example description, otherwise use first
// page description.
```